docs/configuration.md

```diff
@@ -207,8 +209,12 @@ CLIENT_TRANSACTION_RESET= 400000
 CLIENT_DURATION_RESET= 21600
 CLIENT_ERROR_RESET= 100
 MAX_CHUNKS=20
+LOWER_GAS_PRICE_DEVIATION=0.80
+UPPER_GAS_PRICE_DEVIATION=1.20
 ```
 
 > **_NOTE:_** Acceptance tests can be pointed at a remote locations (previewnet and testnet and custom environments). In this case configuration will require details for remote consensus node gRPC endpoints [previewnet / testnet](https://docs.hedera.com/hedera/networks/testnet/testnet-nodes) / [mainnet](https://docs.hedera.com/hedera/networks/mainnet/mainnet-nodes) and [Mirror Node REST API endpoints](https://docs.hedera.com/hedera/sdks-and-apis/rest-api), be sure to configure `HEDERA_NETWORK` and `MIRROR_NODE_URL` appropriately to point away from your local host and to valid deployed services. When pointing to previewnet and testnet, account Ids (`OPERATOR_ID_MAIN`) and private keys (`OPERATOR_KEY_MAIN`) for previewnet and tests may be obtained from the [Hedera Portal](http://portal.hedera.com).
 
 > **_NOTE 2:_**: Read more about `DEV_MODE` [here](./dev-mode.md)
+
+> **_NOTE 3:_**: Hedera transaction fees are set in fiat(USD) but paid using the gas price in HBAR.  The LOWER_GAS_PRICE_DEVIATION and UPPER_GAS_PRICE_DEVIATION allow for a range of gas prices reflecting the current market rates, when testing the current gas price in the acceptance tests.  Read more about transaction fees [here](https://hedera.com/fees)
```

Comment:
Let's add a few lines in here that better explain how this connection can cause the gas price to change on the network.

Reply:
Done.